### PR TITLE
Add endpoint instance show

### PR DESCRIPTION
**Pull request by gabriel-samfira**
Adds the ability to fetch extra details about an instance from a particular endpoint

### CODE REVIEW

File: coriolisclient/cli/endpoint_instances.py [hunk 1]
```diff
@@ -40,6 +43,35 @@ def _get_formatted_data(self, obj):
...
+            "Cores",
+            "OS Type",
+            "Devices",
+            "Firmware type"
+        ]
+
+    def _get_formatted_data(self, obj):
+        data = [
+            obj.id,
+            obj.name,
```
Comment (aznashwan) [inline coriolisclient/cli/endpoint_instances.py:64]: We should ideally print the 'instance_name', as that is the full instance identifier (ex: on vSphere 'instance_name' would be "CI/test/test2/vm" while 'name' would just be 'vm'.

Comment (gabriel-samfira) [reply coriolisclient/cli/endpoint_instances.py:64]: instance name is presented by all providers?

Comment (aznashwan) [reply coriolisclient/cli/endpoint_instances.py:64]: FFS my comment got lost.
Yes, it should be present in all export providers.
I see that it's not in the core JSON schemas, which it absolutely should.

**Review by aznashwan — APPROVED** [review]
LGTM

**Pull request merged**